samples/crypto/rng/README.rst

```diff
@@ -7,14 +7,16 @@ Crypto: RNG
    :local:
    :depth: 2
 
+The RNG sample shows how to produce random numbers.
+
 Overview
 ********
 
-The RNG sample shows how to produce random numbers.
+The sample produces random values of 100-byte length.
```

Comment:
In the overview I don't think details are meritted. What about: This sample shows how to generate random numbers

Reply:
The introdcutory sentence in line 10 already says "The RNG sample shows how to produce random numbers."

So I think here in the first sentence of the overview, it is ok to provide this detail. If not, we just remove it because it doesn't add anything.

Reply:
We can completely remove it later I guess, not a problem.